docs/index.html

```diff
@@ -409,20 +409,24 @@ <h2 class="mb-3">Building the premier multi chain smart contract hub supporting:
                                 </div><!-- .support-info -->
                                 <div class="support-status text-center text-lg-start">
                                   <div class="support-status__item">
-                                    <div class="support-status__number">150K<span class="plus">+</span><span
-                                        class="eth">ETH</span>
+                                    <div class="support-status__number">$2B
                                     </div>
-                                    <div class="support-status__label">TVL Through Lockdrops</div>
+                                    <div class="support-status__label"> Total Value Locked</div>
+                                  </div>
+                                  <div class="support-status__item">
+                                    <div class="support-status__number">$250K
+                                    </div>
+                                    <div class="support-status__label"> Ethereum Assets</div>
                                   </div>
                                   <div class="d-sm-flex d-lg-block justify-content-center">
                                     <div class="support-status__item me-sm-5 me-lg-0">
-                                      <div class="support-status__number">30K<span class="plus">+</span></div>
-                                      <div class="support-status__label"></div>Community Members
+                                      <div class="support-status__number">500K<span class="plus">+</span></div>
+                                      <div class="support-status__label">Community Members</div>
                                     </div>
                                   </div>
                                   <div class="support-status__item">
-                                    <div class="support-status__number">40<span class="plus">+</span></div>
-                                    <div class="support-status__label">Ecosystem Projects</div>
+                                    <div class="support-status__number">6</div>
```

Comment:
Don't think we should display this because it's much more than 6. 
Not possible for us to follow up because we are a permissionless blockchain.